File: codegen/codegen-erlang/src/test/resources/ir/structure_list_decode_encode_item.expected.erl
decode_basic_item_list(undefined) -> undefined;
decode_basic_item_list(null) -> undefined;
decode_basic_item_list(List) when is_list(List) -> [decode_basic_item(V) || V <- List, V =/= null].

encode_basic_item_list(undefined) -> undefined;
encode_basic_item_list(List) when is_list(List) -> [encode_basic_item(V) || V <- List, V =/= undefined].

File: codegen/codegen-elixir/src/test/resources/ir/structure_list_decode_encode_item.expected.ex
defp decode_basic_item_list(nil), do: :nil
defp decode_basic_item_list(list) when is_list(list), do: Enum.map(list, fn v -> decode_basic_item(v) end)

defp encode_basic_item_list(nil), do: :nil
defp encode_basic_item_list(list) when is_list(list), do: Enum.map(list, fn v -> encode_basic_item(v) end)